internal/migration/precheck.go

```diff
@@ -441,7 +441,8 @@ func (ctx *precheckSource) checkModel(stdCtx context.Context) error {
 		return errors.Trace(err)
 	}
 	validators := upgradevalidation.ValidatorsForModelMigrationSource(cloudspec)
-	checker := upgradevalidation.NewModelUpgradeCheck(nil, ctx.backend, model, ctx.modelAgentService, validators...)
+	modelGroupedName := fmt.Sprintf("%s/%s", model.Owner().Id(), model.Name())
```

Comment:
Why we have difference name between `modelNameKey` in `upgrader.go` and `modelGroupedName` here.

Reply:
Good pickup. Next PR coming up will deal with this.